### ISSUE TITLE
Functions like createdOn.Date are ignored in GroupBy

### ISSUE THREAD

**Issue by rivantsov** [rivantsov/vita#226]
See issue #225

**Comment by jasonlaw** [comment]
May I know will this be fixed as well? Thanks.

**Comment by rivantsov** [comment]
yes, it will be fixed as well, will try to focus on this and other general restructuring issues in the next few weeks. I will post more about the plans in a separate post.

**Comment by jasonlaw** [comment]
Thanks for the update, looking into the new release. :)

**Comment by jasonlaw** [comment]
I have one request that would like to get your attention.

Base on the discussion we have before at https://github.com/rivantsov/vita/issues/150, may I know if the filtered relation supported in v1.9 before is part of the sql query or post filtering after retrieving from database? 

If it is part of sql query, may I request the feature again? Here is my use case:
1. A store have many members.
2. The member can unregister themselves from the store, which will be marked as soft deleted.
3. Those soft deleted members will be treated as "deleted record", and should not be retrieved in all the query.
4.  Now, for all the members relationship related query, we always need to add the IsDeleted checking explicitly. It would be nice if this can be handled implicitly by the filtered relation.

Thank you.

**Comment by rivantsov** [comment]
I will definitely look at that, no promises or dates yet, but I am looking at soft deletes
